client/src/components/Loading/index.js

```diff
@@ -14,7 +15,18 @@ const Loading = ({width, height}: Props) => (
     <LoadingComponent type="tail_spin" width={width} height={height} fill="#0062db" />
   </div>
 )
-
+export const EntityDetailLoading = () => (
+  <ListGroupItem className="list-row">
+    <div className="loadingsmall list-row-item">
+      <LoadingComponent type="tail_spin" width={15} height={15} fill="#0062db" />
```

Comment:
Wouldn't the color feel better if it were a constant?

Reply:
Color is now a constant